### ISSUE TITLE
refactor predict_bias.py

### ISSUE THREAD

**Issue by frolovsa** [NOAA-PSL/model_error_correction_with_ai#29]
To accommodate datasets from Stefan, i refactored how the training state is configured (e.g. reduced the number of options and variables stored in the intermediate npy files). As the result, i suspect that https://github.com/NOAA-PSL/model_error_correction_with_ai/blob/main/predict_bias.py is now broken. 

To fix predict_bias.py the following is needed:
- [x] start with the correct branch `https://github.com/NOAA-PSL/model_error_correction_with_ai/tree/feature/feature/stefan_concat_input`
- [x] point to the correct file locations
```
dataDir='/scratch2/BMC/gsienkf/Sergey.Frolov/fromStefan/'   # directory for input data
npyDir=dataDir+'npys_sergey2/ifs_'                            # output directory
```
- [x] note that the order of variables and number of surface fields have changed. see changes to preprocess.py
- [x] point to the correct trained network
`/home/Sergey.Frolov/work/model_error/work/stefan_replay/checks/conv2d_*_4_1_4096_3_0.25_32_mse_0.0001_1.0_366_365_0.7
- [x] point to correct python /scratch1/NCEPDEV/global/Tse-chun.Chen/anaconda3/envs/ltn/bin/python
 
Here is the partial PR that adopts to the new files
https://github.com/NOAA-PSL/model_error_correction_with_ai/pull/31

**Comment by frolovsa** [comment]
See new variables defined here 
https://github.com/NOAA-PSL/model_error_correction_with_ai/blob/903efc52b96b0fd10c86a74ed930f479b0985228/preprocess.py#L22

**Comment by frolovsa** [comment]
closed with #31
